src/codec_impl.rs

```diff
@@ -12,7 +12,9 @@ macro_rules! impl_codec_for {
         impl codec::Decode for $type {
             fn decode<I: codec::Input>(value: &mut I) -> std::result::Result<Self, codec::Error> {
                 let decoded: Vec<u8> = codec::Decode::decode(value)?;
-                bincode::deserialize(decoded.as_slice()).map_err(|_| codec::Error)
+                bincode::deserialize(decoded.as_slice()).map_err(|_| {
+                    codec::Error::from("parity-scale-codec decode error in threshold_crypto")
+                })
```

Comment:
I guess there's no way to include the original error in the message, since `code::Error` expects a static `str` and not a `String`? :pensive:

Reply:
`codec::Error` is an empty struct when built without `std`, maybe the error message is not important here?